apps/snitch_core/lib/core/data/schema/order.ex

```diff
@@ -58,6 +60,7 @@ defmodule Snitch.Data.Schema.Order do
     |> unique_constraint(:number)
     |> foreign_key_constraint(:user_id)
     |> cast_assoc(:line_items, with: &LineItem.create_changeset/2, required: true)
+    |> cast_assoc(:packages)
```

Comment:
No. We never want to changes packages via order.

Reply:
Packages are not updated or edited from Order,
To update total of order, packages are fetched to get the packages costs.

Reply:
To involve `package module` in order to calculate `total` of order.

I didn't see the story - https://www.pivotaltracker.com/n/projects/2149807/stories/158605364

Reply:
If you don't want to edit packages from an order, then why is there a `cast_assoc(:packages)`?